.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "semi": false
```

Comment:
Including this file because my vscode was adding semicolons to the code in the markdown files.